README.md

```diff
@@ -1100,6 +1100,24 @@ If you're already using Python 3.7, switch the `language_version` accordingly. F
 `stable` is a tag that is pinned to the latest release on PyPI. If you'd rather run on
 master, this is also an option.
 
+## GitHub actions
```

Comment:
NIT: actions -> Actions

Reply:
Wasn't sure, since other titles don't capitalise, but this is a proper noun... such hard choices!

Reply:
I based it on how GitHub marketed their service.
https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions

But, in the end, it's just a NIT :)